cfg/auth.py

```diff
@@ -8,7 +8,7 @@
 import json
 import sys
 
-CONFIG_PATH = '../cfg/config.cfg'
+CONFIG_PATH = './cfg/config.cfg'
```

Comment:
This relative import isn't really working out correctly. This might do the trick though. I got it working locally: 

```
import os
dirname = os.path.dirname(__file__)
CONFIG_PATH = os.path.join(dirname, 'config.cfg')
```

Via, https://stackoverflow.com/questions/918154/relative-paths-in-python#918178

Reply:
The issue is that if we install the files as a lib they actually shouldn't be in the same folder as `os.path.dirname(__file__)` which would be the installed library path, no? If the tests are running it means that this import is working in that case. Maybe we need to revise the script cwd?

Reply:
You're right, actually. They should be wherever they're installed. Mine was working because I had it installed via `python setup.py develop`. 

Something is amiss when we run this on the live server and I can't put my finger on it.

Reply:
Here it is, https://github.com/mapswipe/python-mapswipe-workers/blob/eric.p.boucher-fix-imports/cfg/loop_export_module.json#L4

Reply:
Hey,
sorry I got a bit out of touch with this discussion. 

On the server we're running the script in a virtual environment: `/data/environments/mapswipe_workers/bin/python3`

When installing the files as a lib in this virtual environment, we should be able to import them, not?

Reply:
@Hagellach37 Sorry to just now answer your question, yes, we should be able to import them. The problem ended up being that one of the pm2 configs hadn't been updated yet. Eric fixed that.